src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.conditions.txt

```diff
@@ -38,69 +38,53 @@
-   41|       |        // longer treated as a `BasicCoverageBlock` terminator, which changed the coverage region,
-   42|       |        // for the executed `then` block above, to include the closing brace on line 30. That
-   43|       |        // changed the line count, but the coverage code region (for the `else if` condition) is
-   44|       |        // still valid.
-   45|       |        //
-   46|       |        // Note that `if` (then) and `else` blocks include the closing brace in their coverage
-   47|       |        // code regions when the last line in the block ends in a semicolon, because the Rust
-   48|       |        // compiler inserts a `StatementKind::Assign` to assign `const ()` to a `Place`, for the
-   49|       |        // empty value for the executed block. When the last line does not end in a semicolon
-   50|       |        // (that is, when the block actually results in a value), the additional `Assign` is not
-   51|       |        // generated, and the brace is not included.
-   52|      1|        let mut countdown = 0;
-   53|      1|        if true {
-   54|      1|            countdown = 10;
-   55|      1|        }
-   56|       |
-   57|      1|        if countdown > 7 {
-   58|      1|            countdown -= 4;
-   59|      1|        }
-   60|       |        // The closing brace of the `then` branch is now included in the coverage region, and shown
-   61|       |        // as "executed" (giving its line a count of 1 here). Since, in the original version above,
-   62|       |        // the closing brace shares the same line as the `else if` conditional expression (which is
-   63|       |        // not executed if the first `then` condition is true), only the condition's code region is
-   64|       |        // marked with a count of 0 now.
-   65|      0|        else if countdown > 2 {
-   66|      0|            if countdown < 1 || countdown > 5 || countdown != 9 {
-   67|      0|                countdown = 0;
-   68|      0|            }
-   69|      0|            countdown -= 5;
-   70|       |        } else {
-   71|      0|            return;
-   72|       |        }
-   73|      1|    }
-   74|       |
-   75|      1|    let mut countdown = 0;
-   76|      1|    if true {
-   77|      1|        countdown = 1;
-   78|      1|    }
-   79|       |
-   80|      1|    let z = if countdown > 7 {
+   40|      1|        let mut countdown = 0;
+   41|      1|        if true {
+   42|      1|            countdown = 10;
+   43|      1|        }
+   44|       |
+   45|      1|        if countdown > 7 {
+   46|      1|            countdown -= 4;
+   47|      1|        }
+   48|      0|        else if countdown > 2 {
+   49|      0|            if countdown < 1 || countdown > 5 || countdown != 9 {
+   50|      0|                countdown = 0;
+   51|      0|            }
+   52|      0|            countdown -= 5;
+   53|       |        } else {
```

Comment:
Also expecting to see a count here

Reply:
I'm not really sure I follow, but I think the coverage spans are accurate to what is being executed.

Maybe the example in color helps clarify what you're concerned about here? That example also includes the lines you reference in this comment.